pkg/boot/bzimage/header.go

```diff
@@ -207,7 +215,7 @@ type LinuxParams struct {
 	CmdLineSize         uint32               `offset:"0x238"`
 	HardwareSubarch     uint32               `offset:"0x23C"`
 	HardwareSubarchData uint64               `offset:"0x240"`
-	Payload_Ofset       uint32               `offset:"0x248"`
+	PayloadOffset       uint32               `offset:"0x248"`
```

Comment:
I didn't see any references to `Payload_Ofset` in the code, so I renamed it to make lint happy.